crates/ruff_cli/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::print_stdout)]
```

Comment:
Seems bad; required for

```
warning: use of `println!`
    --> /Users/mz/eng/src/astral-sh/ruff/target/debug/build/ruff_cli-05d917f7674e2fe3/out/shadow.rs:1008:2
     |
1008 |     println!("CLAP_LONG_VERSION:{CLAP_LONG_VERSION}\n");
     |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#print_stdout
```

These are for a shadow utility that prints all the variables that we don't use. Will need to investigate a better workaround.